### ISSUE TITLE
Request: Support Properties Catalog Policy Type

### ISSUE THREAD

**Issue by SkipToTheEndpoint** [Micke-K/IntuneManagement#287]
So the new [Enhanced Hardware Inventory](https://techcommunity.microsoft.com/blog/microsoftendpointmanagerblog/enhanced-hardware-inventory-in-intune-coming-in-december/4303744) "Properties Catalog" policy type is now beginning to appear in production tenants, and they exist within at `/beta/deviceManagement/inventoryPolicies`.
It'd be great to be able to export/import these.

I'm having trouble identifying what graph permissions it's using as it just gives me a 403 even though I've given myself everything necessary in the decoded JWT, but your knowledge of API's may fare better than mine :)

**Comment by Micke-K** [comment]
Hello,

I'll have a look. I can't see it my tenant yet. Should show up soon though. 

Cheers!

**Comment by pl4nty** [comment]
It uses the `DeviceManagementConfiguration.ReadWrite.All` delegated scope, but the app scope isn't supported (yet?). Docs were added then later removed, so the interface might not be stable yet

**Comment by Micke-K** [comment]
Thet scope is used on 18 of the APIs. My guess is that they don't allow that API for public IPs yet.
I've added it to the code, but it is not working.

Cheers!

**Comment by mstraessner** [comment]
Is there any current information on this topic? I stumbled across this topic in IntuneManager a few days ago.

**Comment by pl4nty** [comment]
The inventory endpoints block third-party clients, I use portal refresh tokens to get [access tokens with an allowed client ID](https://github.com/pl4nty/intune-change-tracking/blob/b4dd3e3ff780c5e1ed92dacebce0383aea810829/main.py#L205)

**Comment by Micke-K** [comment]
Yes, i don't know why Microsoft has not opened up the api yet. It's a bit frustrating. 

Cheers!

**Comment by roy-klooster-inforcer** [comment]
@Micke-K maybe merging https://github.com/Micke-K/IntuneManagement/issues/294 into this one the opposite would help. Two issues for one topic now
